packages/plugins/deploy-tracker/src/index.js

```diff
@@ -13,6 +13,9 @@ class DeployTracker {
     this.embark.registerActionForEvent("deployment:contract:deployed", trackingFunctions.trackAndSaveContract.bind(trackingFunctions));
     this.embark.registerActionForEvent("deployment:contract:shouldDeploy", deploymentChecks.checkContractConfig.bind(deploymentChecks));
     this.embark.registerActionForEvent("deployment:contract:shouldDeploy", deploymentChecks.checkIfAlreadyDeployed.bind(deploymentChecks));
+    this.embark.registerActionForEvent('scripts-runner:script:executed', trackingFunctions.trackMigrationScript.bind(trackingFunctions));
+
+    this.embark.events.setCommandHandler('deployment-tracker:script:tracked', trackingFunctions.migrationScriptTracked.bind(trackingFunctions));
```

Comment:
We need a command handler to figure out if a certain script is tracked and should therefore be executed or not.

Although, on a second though, it seems like this should be done as `action` instead and have deploy-tracker set a state that `scripts-runner` can rely on.

Reply:
This doesn't feel quite right, it couples the this plugin to another. plugins are not suppose to call actions/events from other plugins.

Also the deploy tracker tracks contracts, while the script plugin requires tracking of scripts. so to solve this, the logic to track the scripts should be implemented in the script plugin itself. This applies both lines 16 and 18